cmd/krel/cmd/gcbmgr.go

```diff
@@ -41,12 +41,16 @@ type gcbmgrOptions struct {
 	releaseType  string
 	buildVersion string
 	gcpUser      string
+
+	nonInteractive bool
 }
 
 var (
 	gcbmgrOpts = &gcbmgrOptions{}
 	buildOpts  = &build.Options{}
 
+	buildPrepareBuilds = build.PrepareBuilds
+	buildRunSingleJob  = build.RunSingleJob
```

Comment:
I would prefer if we move those functions into an interface and mock them via counterfeiter (see the github or git package).

Reply:
ok, cool, i will check in how to do that :) thanks for the feedback